deploy/bundle/manifests/devworkspace-controller-configmap_v1_configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
I really hope that OLM has fixed itself so that there's no issues uninstalling services/configmaps/etc that are stored in the manifests. IIRC in 4.5 there were issues around that

Reply:
Another reason to drop the configmap ;)